### ISSUE TITLE
Tile demo should indicate selection state

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#5891]
**Describe the issue. What is the expected and unexpected behavior?**
Currently when a tile is selected in the demo, the screen reader is not updated on the changed state. This should follow other selected state patterns. I also noticed that the link to the demo section in the documentation is out of date and should be updated.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**
Test the tile demo by screen reader and notice that nothing is announced when tile is selected.

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Demo bug

**Comment by nicolethoen** [comment]
Tiles should set `aria-selected` to true when they are selected. That's likely all that is missing?

**Comment by tlabaj** [comment]
@mcarrano Can we add this one to a milestone please.
